examples/kubernetes/env.sh

```diff
@@ -6,6 +6,11 @@
 # use cases just need KUBECTL=kubectl, we'll make that the default.
 KUBECTL=${KUBECTL:-kubectl}
 
+# Kuberentes api address for $KUBECTL 
+# The default value is 127.0.0.1:8080
+# When the Kuberentes api server is not local, We can easily access the api by edit KUBERENETAPI's value
+KUBERENETAPI=${KUBERENETAPI:-'127.0.0.1:8080'}
```

Comment:
Can you fix the comments to say Kubernetes instead of Kuberentes? Also can you name this var KUBERNETES_API_SERVER to be clearer?

One thing that might help make life simpler in case we need to add any other flags in the future is to create a var in env.sh called KUBECTL_OPTIONS, which would just be '--namespace=$VITESS_NAME --server=$KUBERNETES_API_SERVER' and use that instead in the other scripts. That's optional for this PR though.